practice-app/website/api/participants.py

```diff
@@ -53,13 +53,13 @@ def add_participant(event_id):
 
     
 # API endpoint for removing participants from an event
-@participants.route("/participants/<event_id>", methods=["DELETE"])
+@participants.route("/participants/remove/<event_id>", methods=["POST"])
```

Comment:
I guess you do not check if you selected any participants to remove because it says successfully deleted users even when there are no participants selected. This could be changed to no action

Reply:
You mean on the frontend right, not the api endpoint

Reply:
I thought of the API endpoint to check if participants are empty, but the button in the frontend might be disabled instead if no participants are selected. Both are fine I guess

Reply:
I have added a check to endpoint with commit [5bd0d12](https://github.com/bounswe/bounswe2022group7/pull/232/commits/5bd0d12a4c906588ac4d0356fbf05a377d79bc70). This problem should be solved @azizamankenova

Reply:
yes, it solves the issue now. Thanks, @erimerkin!